modules/dmrpp_module/unit-tests/awsv4_old/get-unreserved/get-unreserved.authz

```diff
@@ -0,0 +1 @@
+AWS4-HMAC-SHA256 Credential=AKIDEXAMPLE/20150830/us-east-1/service/aws4_request, SignedHeaders=host;x-amz-date, Signature=07ef7494c76fa4850883e2b006601f940f8a34d404d0cfa977f52a65bbf5f24f
```

Comment:
Why is this in `awsv4_old`? We have been using `retried` and `unused`  as the directory names for things we move aside. I think this will be super confusing a few months from now.

Reply:
done